lib/error/error_manager_impl.cpp

```diff
@@ -34,10 +34,12 @@ void ErrorManagerImpl::raise_error(const Error& error) {
     if (validate_error_types) {
         if (std::find(allowed_error_types.begin(), allowed_error_types.end(), error.type) ==
             allowed_error_types.end()) {
-            throw EverestArgumentError("Error type " + error.type + " is not allowed to be raised.");
+            EVLOG_error << "Error type " << error.type << " is not allowed to be raised. Ignoring.";
+            return;
```

Comment:
Not sure if we should do not crash here, since the caller isn
t aware of that error isn't raised. In the following the error can't be handled.  Which might lead to critical undefined behaviour..

Reply:
The problem is that it is even possible at compile time to do this. Simply crashing is not well defined behaviour either. So to properly fix this it needs to be detected at compile time. Until then, in my mind ignoring (and logging this in the log) it still better then tearing down the whole system with an uncaught exception.